spec/bolt/inventory_spec.rb

```diff
@@ -748,4 +748,37 @@ def common_data(transport)
         .to raise_error(Bolt::Inventory::ValidationError, /Unsupported version/)
     end
   end
+
+  context 'when using inventory show' do
+    let(:data) {
+      { 'nodes' =>
+                [{ 'name' => 'foo',
+                   'alias' => %w[bar baz],
+                   'config' => { 'ssh' => { 'disconnect-timeout' => 100 } },
+                   'facts' => { 'foo' => 'bar' } }] }
+    }
+
+    let(:inventory) { Bolt::Inventory.new(data) }
+    let(:target) { get_target(inventory, 'foo') }
+    let(:expected_data) {
+      { 'name' => 'foo',
+        'alias' => %w[bar baz],
+        'config' =>
+                          { 'transport' => 'ssh',
```

Comment:
This is what rubocop wants for spacing. Is it worth trying to find a way around it???

Reply:
I don't think so - doesn't seem worth the time.